scripts/prepare.sh

```diff
@@ -25,15 +25,11 @@ npx jetify
 # Copy the API key to native code:
 node -r sucrase/register ./scripts/makeNativeHeaders.ts
 
-# Create zcash checkpoints
-mkdir -p android/app/src/main/assets/saplingtree/mainnet
-cp -R node_modules/edge-currency-accountbased/lib/zcash/zecCheckpoints/ android/app/src/main/assets/saplingtree/mainnet 2>/dev/null || :
-cp -R node_modules/edge-currency-accountbased/lib/zcash/zecCheckpoints/ ios/Pods/ZcashLightClientKit/Sources/ZcashLightClientKit/Resources/saplingtree-checkpoints/mainnet 2>/dev/null || :
+# Update zcash checkpoints for iOS
+cp -r node_modules/edge-currency-accountbased/android/src/main/assets/piratesaplingtree/mainnet ios/Pods/ZcashLightClientKit/Sources/ZcashLightClientKit/Resources/saplingtree-checkpoints/mainnet
```

Comment:
The paths are switched between pirate and zcash. Replace piratesaplingtree with saplingtree in the first path